packages/advanced-logic/test/extensions/payment-network/meta.test.ts

```diff
@@ -0,0 +1,372 @@
+import { ExtensionTypes, RequestLogicTypes } from '@requestnetwork/types';
+import { deepCopy } from '@requestnetwork/utils';
+import { CurrencyManager, UnsupportedCurrencyError } from '@requestnetwork/currency';
+
+import * as DataConversionERC20FeeAddData from '../../utils/payment-network/erc20/any-to-erc20-proxy-add-data-generator';
+import * as MetaCreate from '../../utils/payment-network/meta-pn-data-generator';
+import * as TestData from '../../utils/test-data-generator';
+import MetaPaymentNetwork from '../../../src/extensions/payment-network/meta';
+
+const metaPn = new MetaPaymentNetwork(CurrencyManager.getDefault());
+const baseParams = {
+  feeAddress: '0x0000000000000000000000000000000000000001',
+  feeAmount: '0',
+  paymentAddress: '0x0000000000000000000000000000000000000002',
+  refundAddress: '0x0000000000000000000000000000000000000003',
+  salt: 'ea3bc7caf64110ca',
+  network: 'rinkeby',
+  acceptedTokens: ['0xFab46E002BbF0b4509813474841E0716E6730136'],
+  maxRateTimespan: 1000000,
+} as ExtensionTypes.PnAnyToErc20.ICreationParameters;
+
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+describe('extensions/payment-network/meta', () => {
+  describe('createCreationAction', () => {
+    it('can create a create action with all parameters', () => {
+      expect(
+        metaPn.createCreationAction({
+          [ExtensionTypes.PAYMENT_NETWORK_ID.ANY_TO_ERC20_PROXY]: [baseParams, baseParams],
+        }),
+      ).toEqual({
+        action: 'create',
+        id: ExtensionTypes.PAYMENT_NETWORK_ID.META,
+        parameters: {
+          [ExtensionTypes.PAYMENT_NETWORK_ID.ANY_TO_ERC20_PROXY]: [baseParams, baseParams],
```

Comment:
Is it OK to have two with the same salt? Shouldn't it throw?

Reply:
Nice catch.
This validation was only in the `applyAction` logic. It's now in the `createCreationAction` logic as well. I added a new test to reflect that.